docs/dev/vagrant.rst

```diff
@@ -49,20 +54,24 @@ Next, use Vagrant to create your VM: ::
 
     vagrant up
 
+(If you get an error message suggesting that you run ``vagrant init``, you probably forgot to ``cd devsite``.)
+
 Note that you will not be able to see the VM, since it runs in a "headless" mode by default.
 
 The following command connects to the running VM and installs the software dependencies: ::
 
     fab setup
 
-The development environment can be seeded with a database dump from an existing chapter, subject to a confidentiality agreement and security requirements on the part of the developer.  The ``loaddb`` task accepts an optional argument to load a database dump in .sql or any other Potgres-supported dump format. ::
+The development environment can be seeded with a database dump from an existing chapter, subject to a confidentiality agreement and security requirements on the part of the developer.  The ``loaddb`` task accepts an optional argument to load a database dump in .sql or any other Potgres-supported dump format. (**NOTE:** Run **only one** of the next three commands.) ::
 
     fab loaddb:/path/to/dump
 
 Alternatively, database dumps can be downloaded automatically over HTTP. If you've been provided with a download URL, run: ::
```

Comment:
I think you should edit the documentation to say this is the preferred method if you are given a download link.